framework/src/utils/RankTwoTensor.C

```diff
@@ -929,29 +929,20 @@ RankTwoTensorTempl<T>::symmetricEigenvaluesEigenvectors(std::vector<T> & eigvals
 
 template <typename T>
 RankTwoTensorTempl<T>
-RankTwoTensorTempl<T>::rowPermutationTensor(std::vector<unsigned int> old_row,
-                                            std::vector<unsigned int> new_row) const
+RankTwoTensorTempl<T>::permutationTensor(std::vector<unsigned int> old_elements,
+                                         std::vector<unsigned int> new_elements) const
 {
-  mooseAssert(N == old_row.size() && N == new_row.size(),
-              "rowPermutationTensor: number of rows should be equal to LIBMESH_DIM");
+  mooseAssert(N == old_elements.size() && N == new_elements.size(),
+              "permutationTensor: number of permuting elements should be equal to LIBMESH_DIM");
```

Comment:
The check and the assertion message are not checking/describing the same thing here